litert/vendors/intel_openvino/compiler/graph_iterator.cc

```diff
@@ -0,0 +1,99 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "graph_iterator.h"
+
+namespace litert {
+namespace openvino {
+size_t GraphIteratorDelegate::size() const {
+    return iterator_indices_.input_index_ + iterator_indices_.output_index_ +
```

Comment:
I see some of these are missing have TODO's in the first commit but addressed here . Can we clean it up to have a clean commit in the history - its ok to have multiple commits but avoid commented/unused code in history